doc/cephfs/dirfrags.rst

```diff
@@ -46,7 +46,7 @@ Size thresholds
 ===============
 
 A directory fragment is eligible for splitting when its size exceeds
-``mds_bal_split_size`` (default 10000).  Ordinarily this split is
+``mds_bal_split_size`` (default 10000 omap entries).  Ordinarily this split is
```

Comment:
Is this number of the directory entries or omap entries ?

The fragment size returns the sum of files and subdirs ?
```
struct frag_info_t : public scatter_info_t {
  int64_t size() const { return nfiles + nsubdirs; }

```

Reply:
Should this be a :confval: reference?

Reply:
@anthonyeleven The aim of this change is to clarify what `10000` here means. Even if we use :confval: reference, we still would need to mention the unit of what `10000` here is. I opine appending the unit to this value would be enough. What say?

Reply:
@kotreshhr Should I then use 10000 (sum of files and subdirs) instead of omap entries?

Reply:
I prefer 'directory entries' or 'entries'.

Reply:
s/omap entries/directory entries done. PTAL. Also applies to https://github.com/ceph/ceph/pull/46950#discussion_r914543553 and https://github.com/ceph/ceph/pull/46950#discussion_r914543961.